### DISCUSSION TITLE
How is the c++ websocket implementation here related to the implementation in the foxglove rosbridge

### DISCUSSION THREAD

**Discussion by jlack1987** [foxglove/community#1035] [Q&A]
I've recently been working on using the foxglove websocket protocol so I can send log data to foxglove studio and I found this implementation that i've been using, but I also just noticed that there seems to be a parallel implementation [here](https://github.com/foxglove/ros-foxglove-bridge) and i'm just wondering which one is the most up-to-date/maintained. Seems if it's implemented in 2 places one could be removed and just add a dependency on the other instead.

**Comment by jlack1987** [comment]
Actually I think I was wrong and the rosbridge repo seems to be the one that's most up to date

**Reply by achim-k** [reply]
Both implementations should be very up to date & maintained
